src/index.html

```diff
@@ -3,11 +3,281 @@
+              >
+            </label>
+            <label class="form__item" for="phone">
+              <input
+                class="input form__input"
+                type="text"
+                name="phone"
+                placeholder="Phone"
+                id="phone"
+                required
+              >
+            </label>
+            <label class="form__item" for="email">
+              <input
+                class="input form__input"
+                type="text"
+                name="email"
+                placeholder="Email"
+                id="email"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
+                required
+              >
+            </label>
+            <button class="button form__button" type="submit">Consultation</button>
+          </form>
+        </div>
+      </section>
+
+      <section class="section">
+        <h2 class="heading section__heading" id="testimonials">
+          Testimonials
+          <span class="heading__crown"></span>
+        </h2>
+        <div class="section__testimonials testimonials">
+          <div class="person">
+            <img
+              class="person__photo"
+              src="./images/people/1.jpg"
+              alt="Leslie Mckinney"
+            >
+            <h3 class="person__name">Leslie Mckinney</h3>
+            <a class="person__position" href="https://goo.gl/maps/iedU6KSKWzwwvj1G9">CEO</a>
+            <p class="person__text">Nice work, Certificates National Association of Realtors (USA)</p>
+          </div>
+          <div class="person">
+            <img
+              class="person__photo"
+              src="./images/people/2.jpg"
+              alt="Scarlett Hawkins"
+            >
+            <h3 class="person__name">Scarlett Hawkins</h3>
+            <a href="https://goo.gl/maps/iedU6KSKWzwwvj1G9" class="person__position">Senior Vice President</a>
+            <p class="person__text">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart</p>
+          </div>
+          <div class="person">
+            <img
+              class="person__photo"
+              src="./images/people/3.jpg"
+              alt="Max Henry"
+            >
+            <h3 class="person__name">Max Henry</h3>
+            <a class="person__position" href="https://goo.gl/maps/iedU6KSKWzwwvj1G9">Chief Design Officer</a>
+            <p class="person__text">When my lovely valley of steam rises and half-day sun is above an impermeable</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section">
+        <h2 class="heading section__heading" id="contacts">
+          Contact us
+          <span class="heading__crown"></span>
+        </h2>
+        <div class="section__contacts contacts">
+          <div class="contacts__text">
+            <div>
+              <p>Phone<br><a href="tel:+12345555555" class="contacts__link">+1 234 5555-55-55</a></p>
```

Comment:
```suggestion
              <p>
                Phone<br>
                <a href="tel:+12345555555" class="contacts__link">+1 234 5555-55-55</a>
              </p>
```

Форматуй будь ласка код, не залишай довгі строки в один рядок